### PR TITLE
[JSX] Refactor and Update Pagination

### PR DESCRIPTION
**Pull request by HenriRabalais**
Brief summary of changes
This PR refactors the Pagination component by changing it from a class component to a functional component that utilizes React Hooks. It also removes the bootstrap styling, replacing it with entirely custom dependency-free styling.

Display
112882516-11ece300-909b-11eb-83b4-955cd756c8e1
112882525-144f3d00-909b-11eb-87f1-d8a790ec41b5

Grander Vision
This is the first PR in a series that I am planning to propose over the coming months. This is with the main objective of refactoring much of the now outdated jsx code found in LORIS. In the process of refactoring, I would like to accomplish the following:

Convert React components from Class Components to Functional Components that use Hooks state management
Functional components and Hooks are the lastest in React development, but have now been out for several years and have had time to be rigourously tested and seem to be very encouraged by the community. From my use of them, I find they tend to lead to simplified code that results in less lines and increased legibility.
References:
-- Class vs. Functional: https://www.twilio.com/blog/react-choose-functional-components)
-- React Hooks: https://reactjs.org/docs/hooks-intro.html
Reduce our reliance on Bootstrap Boilerplate
Bootstrap was a good launching pad for LORIS to develop a cohesive design aesthetic. However, the bootstrap "look" is both outdated and also leads to stylesheet graveyards as we (rightfully) begin to overwrite the original bootstrap classes with our own custom css. However, I believe keeping the formatting inline with the React Components will increase our ability to quickly access and make changes to css when necessary since they will be found within the same file.
This will also make it easier to have global "themes" that permeate through LORIS and will be easily changeable from a single location (e.g. dark vs light themes). I believe this to be a much more modern approach to styling LORIS which will result in less time wasted looking and sorting through style sheets and make customizability and quick changes easier.
Incorporate New Components
In the creation of the Biobank module, I've added many components and made alterations to ones that currently exist in LORIS. In the process of refactoring the LORIS Components, I would like to both include my modification as well as incorporate the new components that I've written.
Create a Design Theory for LORIS
It's no secret that LORIS has inconsistence when it comes to the UI throughout the platform. I am hoping that through this process, a series of standardized layouts components can be created, accompanied by a design "theory" for their use that will act as guidelines for how modules and pages within LORIS should be organized to maintain a consistent and coherent user experience.
This will include specific sets of panels, buttons, fields, menus, modals, dropdowns etc. that have a unified design with the flexibility of the users needs in mind (e.g. using 'rem' instead of 'px' and making the webpage mobile and tablet friendly).
Reference for use of REM: https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -20,4 +20,5 @@ htdocs/js/FileSaver.min.js
 htdocs/vendor/
 htdocs/fontawesome/
 htdocs/bootstrap/
-
+project/modules2/
```
Comment (driusan) [inline .eslintignore:23]: These changes seem unrelated

**Comment by cmadjar** [comment]
@HenriRabalais can you please fix the tests and integrate Dave's comment?

@jesscall could you please review this once it passes the tests?

**Review by maltheism — APPROVED** [review]
@driusan see my previous comment.

**Comment by ridz1208** [comment]
will be reopened by Henri if needed
